fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/OcflPersistentStorageSession.java

```diff
@@ -434,9 +427,19 @@ private void actionNeedsWrite() throws PersistentStorageException {
         }
     }
 
-    private void cleanupStagingDir() {
-        if (Files.exists(sessionStagingDir) && !FileUtils.deleteQuietly(sessionStagingDir.toFile())) {
-            LOGGER.warn("Failed to cleanup session staging directory at {}", sessionStagingDir);
+    /**
+     * Returns the RDF topic to be returned for a given resource identifier
+     * For example:  passing info:fedora/resource1/fcr:metadata would return
+     *  info:fedora/resource1 since  info:fedora/resource1 would be the expected
+     *  topic.
+     * @param fedoraIdentifier The fedora identifier
+     * @return The resolved topic
+     */
+    private FedoraId resolveTopic(final FedoraId fedoraIdentifier) {
+        if (fedoraIdentifier.isDescription()) {
```

Comment:
Should we also be looking for other "child" types? such as, ACL or HashURI?
..or, is there any reason not to also return the `asBaseId()`?

Reply:
To be honest, I don't understand the behavior here and was just keeping it consistent with the existing code. I'm more than happy to change it, but I don't know hat the correct behavior is.